jupyterhub_singleuser_profiles/service.py

```diff
@@ -17,17 +17,11 @@
 _REFERENCE_CM_NAME = "singleuser-service-ref-%s"
 
 class Service():
-  def __init__(self, server_url, token, namespace=None, verify_ssl=True):
-    self.server_url = server_url
-    self.token = token
+  def __init__(self, namespace, verify_ssl=True):
     service_account_path = '/var/run/secrets/kubernetes.io/serviceaccount'
 
-    if namespace:
-      self.namespace = namespace
-    else:
-      with open(os.path.join(service_account_path, 'namespace')) as fp:
-          self.namespace = fp.read().strip()
-
+    self.namespace = namespace
+    print(self.namespace)
```

Comment:
Did you intentionally leave in this print statement?

Reply:
Nope, fixed